contracts/src/v0.8/keystone/KeystoneFeedsConsumer.sol

```diff
@@ -88,8 +88,8 @@ contract KeystoneFeedsConsumer is IReceiver, ConfirmedOwner {
     // workflow_owner           // offset 74, size 20
     // report_name              // offset 94, size  2
     assembly {
-      // shift right by 22 bytes to get the actual value
-      workflowName := shr(mul(22, 8), mload(add(metadata, 64)))
+      // no shifting needed for bytes10 type
+      workflowName := mload(add(metadata, 64))
```

Comment:
Ah right, I fixed this in the forwarder but didn't realize there's a matching snippet in the consumer